### PR TITLE
R&R: add photos & scrolling to reviews, reorganize files

### PR DESCRIPTION
**Pull request by davidguy3237**

### CODE REVIEW

File: client/src/components/ratings-and-reviews/reviews/ReviewsList.jsx [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+        },
+      })
+        .then(({ data }) => setReviews(data))
+        .then(() => axios.get('/reviews', {
+          params: {
+            page: 2,
+            product_id: product.id,
+          },
+        }))
+        .then(({ data }) => setNextPage(data));
```
Comment (beanjermin) [inline client/src/components/ratings-and-reviews/reviews/ReviewsList.jsx:26]: Add catch statement

File: client/src/components/ratings-and-reviews/reviews/ReviewsList.jsx [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+        page,
+        product_id: product.id,
+      },
+    })
+      .then(({ data }) => {
+        setReviews((currentReviews) => [...currentReviews, ...nextPage]);
+        setNextPage(data);
+        setPage(page + 1);
+      });
+  };
```
Comment (beanjermin) [inline client/src/components/ratings-and-reviews/reviews/ReviewsList.jsx:42]: add catch

**Review by beanjermin — APPROVED** [review]
heyyy looks good! I noticed you're missing a couple catches at the end of your http requests. Ima approve, but make sure to catch your errors!

File: client/src/components/ratings-and-reviews/reviews/ReviewsList.jsx (hunk 1 above)
Comment (davidguy3237) [reply client/src/components/ratings-and-reviews/reviews/ReviewsList.jsx:26]: will do!

**Pull request merged**